ruma-federation-api/src/transactions/send_transaction_message/v1.rs

```diff
@@ -22,15 +22,19 @@ ruma_api! {
         /// A transaction ID unique between sending and receiving homeservers.
         #[ruma_api(path)]
         pub transaction_id: String,
+
         /// The server_name of the homeserver sending this transaction.
         pub origin: String,
-        /// POSIX timestamp in milliseconds on originating homeserver when this transaction started.
+
+        /// POSIX timestamp in milliseconds on the originating homeserver when this transaction started.
```

Comment:
Is this taken from the Matrix docs by the way? I don't think I've seen it referred to as POSIX timestamp before, only as time since UNIX epoch.

Reply:
Yeah, looks like I copied it straight from there.